.travis.yml

```diff
@@ -6,7 +6,7 @@ matrix:
     before_install: cd Node
   - &py-ci
     language: python
-    python: '2.7'
+    python: '3.7'
```

Comment:
@torsstei May I have your confirmation if it plans to drop support on Python 2.7 for this SDK? (I'm happy to do this but it really depends on whether there is any customer need it)

Reply:
@shawnzhu I am not aware of a customer who explicitly requires Python 2.7. Maybe some are out there using it with 2.7, but we can invoke the fact that 2.7 itself went out of support. If a customer would still insist for us to bring back 2.7 support and can make the business case to us, we would then see to bring it back somehow. But after all it is open source and such customer could also just fork and patch or engage in 2.7 support maintenance in the repo here.